quickstart/verify.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+temp_file=$(mktemp)
+
+curl http://localhost:8888/api/v1alpha1/meters/m1/values?windowSize=HOUR >$temp_file
+
+[ $(cat $temp_file | jq '.data[0].value') == 2 ] || (echo "Unexpected value" && cat $temp_file && exit 1)
```

Comment:
We should probably verify all the fields, except `windowStart` and `windowEnd`.

Reply:
PRs welcome in a follow up PR. ;)